software/minus-games.yml

```diff
@@ -0,0 +1,11 @@
+# software name
+name: "Minus Games"
+website_url: "https://github.com/Accessory/minus_games"
+source_code_url: "https://github.com/Accessory/minus_games"
+description: "Server and launcher combo that helps you sync game and save files across multiple devices."
```

Comment:
I don't understand what the software does, and the README doesn't help much. Can you point me to a doc explaining what the features are, what the projects' goals/primary use cases are, etc?

Reply:
More infos and a Guide can be found @ https://accessory.github.io/minus_games_user_guide/